frontend/src/components/samples/SamplesListContent.tsx

```diff
@@ -140,12 +140,12 @@ function SamplesListContent() {
 					/>
 					<FiltersBar filters={samplesTableState.filters} clearFilters={clearFiltersAndCategory}/>
 				</Flexbar>
-				
-				<PagedItemsTable<ObjectWithSample> 
+				<PagedItemsTable<ObjectWithSample>
 					getDataObjectsByID={mapSampleIDs}
 					pagedItems={samplesTableState}
 					columns={baseColumns}
 					usingFilters={false}
+					initialLoad={false}
```

Comment:
Not sure we should making a patch is the way to go. I would just add back the missing code and either fix the underlying issue or let it double query ...